apis/kueue/webhooks/clusterqueue_webhook.go

```diff
@@ -108,6 +111,14 @@ func ValidateClusterQueue(cq *kueue.ClusterQueue) field.ErrorList {
 	return allErrs
 }
 
+// We need to do this before Kubernetes 1.25.
+func ValidateClusterQueueUpdate(newObj, oldObj *kueue.ClusterQueue) field.ErrorList {
+	var allErrs field.ErrorList
+	allErrs = append(allErrs, apivalidation.ValidateImmutableField(newObj.Spec.QueueingStrategy, oldObj.Spec.QueueingStrategy, field.NewPath("spec", "queueingStrategy"))...)
```

Comment:
Remove the comment above and add here:
// We need to validate the spec.queueingStrategy immutable manually before Kubernetes 1.25.